### DISCUSSION TITLE
en/chapter_introduction/algorithms_are_everywhere/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1049] [Announcements]
# en/chapter_introduction/algorithms_are_everywhere/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_introduction/algorithms_are_everywhere/

**Comment by ankit262** [comment]
Flow based algorithm

**Reply by ankit262** [reply]
Hi

**Comment by fgdhs788687** [comment]
hello, very good explanation is done by you many many thanks it helps a lot.

**Comment by tran-huy-phuc** [comment]
I studied DSA in university 17-18 years ago. Now I almost forget them (that's why I'm here). You give great examples!
